### DISCUSSION TITLE
Cloud usage aws

### DISCUSSION THREAD

**Discussion by 19167458** [Security-Onion-Solutions/securityonion#2272] [Unsupported Versions] [answered]
Thank you for this amazing product. I had a question regarding this product, can I install this on the cloud and monitor an organisation and its local servers? Or do I need to deploy it locally. I want to run this on the cloud and use it to monitors local servers. I am unable to find proper documentation related to this.

**Comment by DashCarlos** [comment]
I am a newbie but I think that redirecting all the traffic of that organization to a node in AWS is not a good option, at least the Standalone deployment is a bad option, the Distributed mode may not pose much problem, you would have the management node in AWS and the forward nodes should be in the network of the organization you want to monitor

**Comment by weslambert** [comment, marked as answer]
Folks typically run:
(1) Manager only in AWS
(2) Whole distributed deployment in AWS, to monitor cloud-only traffic
(3) Sensors only in AWS to monitor-cloud traffic

**Reply by hirenshah005** [reply]
Can you please elaborate a little more? Can I monitor all EC2 instances network traffic, VPC traffic, and other all components traffic? any docs or helpful videos will help.

**Reply by weslambert** [reply]
You could only monitor Nitro-based EC2 instances via the traffic mirroring capability.
